### ISSUE TITLE
Items vanish from "new", but are removed for good

### ISSUE THREAD

**Issue by holgerschurig** [hacs/integration#4238]
### Web browser

Firefox

### Web browser version

132.0.1

## System Information

version | core-2024.11.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.4
os_name | Linux
os_version | 6.6.54-haos
arch | x86_64
timezone | Europe/Berlin
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
HACS Data | ok
GitHub API Calls Remaining | 4950
Installed Version | 2.0.1
Stage | running
Available Repositories | 1468
Downloaded Repositories | 12

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 13.2
-- | --
update_channel | stable
supervisor_version | supervisor-2024.11.4
agent_version | 1.6.0
docker_version | 27.2.0
disk_total | 30.8 GB
disk_used | 9.1 GB
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | kvm
board | ova
supervisor_api | ok
version_api | ok
installed_addons | Advanced SSH & Web Terminal (19.0.0), Mosquitto broker (6.4.1), rtl_433 (0.5.2), Whisper (2.2.0), Piper (1.5.2), Zigbee2MQTT (1.41.0-1), MQTT Explorer (browser-1.0.3), Tailscale (0.23.3), VLC (0.3.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 2
-- | --
resources | 5
views | 8
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | November 28, 2024 at 10:55 AM
-- | --
current_recorder_run | December 1, 2024 at 12:04 PM
estimated_db_size | 144.52 MiB
database_engine | sqlite
database_version | 3.45.3

</details>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/help/issues/>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This is a bug and not a feature request.
- [X] This issue is not a duplicate issue of currently [open](https://github.com/hacs/integration/issues) or issues [pending release](https://github.com/hacs/integration/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+milestone%3Anext).

### Describe the issue

Items from the "New" section vanish. And it is unclear

a) how "New" is maintained
b) where vanished items are going when removed from "New"

On a larger scale, "New" is completely puzzling, since it's entirely unclear how items make into new and how they are removed. I think it's also totally superfluous, as the items could as well be put into "Available for Download". If one sorts by "Last activity" then one would see new items there: the download count would reveal if it's new or not.

### Reproduction steps

1. Select HACS on the sidepanel
2. Scroll down to the "New" section
3. if it's currently closed, open it
4. look at the two topmost entries, currently "Ha Beringer Mixer" and "Ha Starlinghomehub" for me
5. click on, I used the Beringer one
6. the Github page opens
7. press the "back" arrow of the browser
8. Notice that the entry has now vanished, the top-most entry is no "Ha Starlinghomehub"
9. Notice that the entry isn't visible at "Available for Download" either
10. Notice that if you search for "Beringer" you won't get any hit

So basically the component has vanished. But more:

11. Click on "Calender" (or something else, to get away from the HACS view)
12. Click again on "HACS"
13. "Beringer" is still nowhere to be seen. And it is still no searchable.

Currently I'm at a loss on how to get back these vanished items.

### Screenshots

No screenshot needed I'd say.

### Javascript logs from your browser console

```text
Honestly asking for Javascript logs is dump. I refuse.
```

**Comment by ludeeus** [comment]
"New" are new to you, as soon as you open them, they are no longer new to you.
